pkg/syncer/syncer.go

```diff
@@ -59,6 +66,23 @@ const (
 	heartbeatInterval = 20 * time.Second
 )
 
+type FilteredGVRSource struct {
```

Comment:
public? Why at the top of the most central file of the syncer?

Reply:
Right. renamed it and moved it to the bottom.